lib/ansible/plugins/inventory/tower.py

```diff
@@ -20,6 +20,10 @@
           the path in the command would be /path/to/tower_inventory.(yml|yaml). If some arguments in the config file
           are missing, this plugin will try to fill in missing arguments by reading from environment variables.
         - If reading configurations from environment variables, the path in the command must be @tower_inventory.
+    deprecated:
+        alternative: Use U(https://galaxy.ansible.com/awx/awx) instead.
```

Comment:
This is a good test change for the new docs pipeline. Can we hold this PR until #59761 is ready?

Reply:
That PR looks to be correspondingly blocked on https://github.com/ansible/galaxy/issues/1992

I see that in the foreman case they have their own docs hosting solution, and that's something that we have no intention to bother with for the AWX collection. So it makes sense to me push for the solution there.